client/a8c-for-agencies/components/form/footer/style.scss

```diff
@@ -1,11 +1,30 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
 .a4a-form__footer {
 	margin-block-start: 8px;
 	padding-block: 24px;
 	border-block-start: 1px solid #C3C4C7;
-
 	display: flex;
-	gap: 24px;
+	flex-direction: column;
+	gap: 8px;
+
+	div {
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: 8px;
+	}
+
+	@include break-small {
+		flex-direction: row;
+		gap: 24px;
+		justify-content: space-between;
+		align-items: center;
 
-	justify-content: space-between;
-	align-items: center;
+		div {
+			display: block;
+			align-items: unset;
+		}
```

Comment:
Should we move this outside from the shared footer style?

This look like specific to the signup page.

Reply:
Agree. Fixed here: https://github.com/Automattic/wp-calypso/pull/100128/commits/9825dc22b5285c7c75b36cfd98a5ebf46f9f48f3